roles/ovirt-infra/README.md

```diff
@@ -80,7 +85,10 @@ Example Playbook
      
      data_center_name: mydatacenter
      compatibility_version: 4.1
-     
+
+     mac_pool_allow_duplicates: false
```

Comment:
you mean ovirt-mac-pool/README.md ? 
there it is also mac_pool_allow_duplicates